lib/src/room/rooms.dart

```diff
@@ -75,33 +76,52 @@ class Rooms extends DelegatingIterable<Room> implements Contextual<Rooms> {
       return this;
     }
 
-    return copyWith(
-      rooms: List.of([
-        // Merge all rooms that are present in this as well as in other
-        ...map((room) {
-          Room? otherRoom;
-
-          final otherRooms = other.where(
-            (otherRoom) => otherRoom.equals(room),
-          );
-
-          if (otherRooms.isEmpty) {
-            return room;
-          } else if (otherRooms.length == 1) {
-            otherRoom = otherRooms.first;
-          } else {
-            for (final anotherRoom in otherRooms) {
-              otherRoom = otherRoom?.merge(anotherRoom) ?? anotherRoom;
-            }
-          }
-
-          return room.merge(otherRoom!);
-        }),
-        // All other rooms that were not merged, they're new
-        ...other.where(
-          (otherRoom) => !any((room) => otherRoom.equals(room)),
-        ),
-      ], growable: false),
+    final currList = toList();
```

Comment:
Is that legal wrote just "toList"?

Reply:
mean this.toList(). Short version because dart analysis.

Reply:
toList is internal method for Rooms Model

Reply:
Understood, thanks